build_submodules.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+cd ./src/submodules/js-ceramic
+npm install
+cd ./packages/ipfs-daemon
+npm install
```

Comment:
I don't think this second npm install should be necessary.  Also not sure if it's harmful but maybe could be?  idk lerna is weird

Reply:
👍